_posts/2016-12-07-forget-the-keyboard.md

```diff
@@ -45,7 +45,7 @@ When your fingers are on the home row, you know that if you type with *that* fin
 
 ![Position of letter j](static/img/hands-keyboard-j.jpg)
 
-Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. **Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
+Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>**Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
```

Comment:
Ok, I like to put this in a new line, but still in the same paragraph. Good hint, thank you.